packages/ui/client/components/views/ViewModuleGraph.vue

```diff
@@ -106,6 +126,14 @@ html.dark {
   --color-node-inline: #468b60;
 }
 
+.graph {
+  min-height: var(--graph-h) !important;
+  max-height: var(--graph-h) !important;
+  height: var(--graph-h) !important;
+  min-width: var(--graph-w) !important;
+  max-width: var(--graph-w) !important;
+  width: var(--graph-w) !important;
+}

```

Comment:
You can safely remove the style and ResizeObserver changes.

The graph already uses the size of the container, as the implementation of `resize` shows below:

```ts
public resize(): void {
    const newWidth = this.container.offsetWidth
    const newHeight = this.container.offsetHeight
    const widthDiffers = this.width.toFixed() !== newWidth.toFixed()
    const heightDiffers = this.height.toFixed() !== newHeight.toFixed()

    if (!widthDiffers && !heightDiffers) {
      return
    }

    this.resetView()
    this.initGraph()
    this.restart(this.config.alphas.resize)
}
```

Reply:
https://imgur.com/XIAzdUu

Reply:
This isn't present in the current version, so it has to be caused by one of the changes.

Reply:
I'm 99% sure that this is caused by

```ts
const data = asyncComputed(async() => {
  const filepath = current.value?.filepath
  return filepath
    ? await client.rpc.getModuleGraph(filepath)
    : { externalized: [], graph: {}, inlined: [] }
})
```

re-emitting every time tests are run.

Either that or the entire `FileDetails` are recreated when the tests are rerun.

Reply:
ok, I'll review that code...

Reply:
fixed